pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -45,7 +41,11 @@ class Meta:
         fields = ['name', 'name_in_list']
 
 
-class RepositoryViewSet(NamedModelViewSet):
+class RepositoryViewSet(NamedModelViewSet,
+                        mixins.CreateModelMixin,
+                        mixins.UpdateModelMixin,
```

Comment:
Despite defining ```update()```, the CRUD tests fail without the UpdateModelMixin because UpdateModelMixin is defining ```partial_update()```

Is that OK?  I don't think we want it to be able to partially update without going through our tasking system.  And we should check other spots for this, too.

Reply:
Cool